### PR TITLE
Add Drawer for showing critical facility details

### PR DESCRIPTION
**Pull request by Mloweedgar**
- [x] add Drawer for showing critical facility details
- [x] add  Header component for showing critical facility header
- [x]   add  Body component for showing critical facility Body details

### CODE REVIEW

File: src/GeographicalFeatures/CriticalFacilities/index.js [hunk 1]
```diff
@@ -377,6 +384,30 @@ class FeatureList extends Component {
...
+  };
+
+  /**
+   * @function
+   * @name closeDetails
+   * @description close feature details drawer
+   * @version 0.1.0
+   * @since 0.1.0
+   */
+  closeDetails = () => {
```
Comment (BenMaruchu) [inline src/GeographicalFeatures/CriticalFacilities/index.js]: Hello @Mloweedgar, Thanks for this PR.

Can you prefix these handler function with `handle` like what you  did with the handleView?

File: src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/**
+ * @function
+ * @name CriticalFacilityDetailsViewBody
+ * @description details of a critical facility
+ * @returns {object} React component
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+const CriticalFacilityDetailsViewBody = () => {
+  return <div>This is body</div>;
```
Comment (BenMaruchu) [inline src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js]: Is this still `Work in Progress`?

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js (hunk 2 above)
Comment (Mloweedgar) [reply src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js]: Yes. I wanted to give you a chance to look at my direction, also I did not want to send a huge PR with  many files changed

File: src/GeographicalFeatures/CriticalFacilities/DetailsView/Header.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    onBack={() => onBack()}
+  />
+);
+
+export default CriticalFacilityDetailsViewHeader;
+CriticalFacilityDetailsViewHeader.propTypes = {
+  name: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  onBack: PropTypes.func.isRequired,
+};
```
Comment (BenMaruchu) [inline src/GeographicalFeatures/CriticalFacilities/DetailsView/Header.js:31]: can we move this block above config export to ensure consistence

File: src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  }).isRequired,
+  onEdit: PropTypes.func.isRequired,
+};
+
+CriticalFacilityToolbar.propTypes = {
+  criticalFacility: PropTypes.shape({
+    _id: PropTypes.string,
+  }).isRequired,
+  onEdit: PropTypes.func.isRequired,
+};
```
Comment (BenMaruchu) [inline src/GeographicalFeatures/CriticalFacilities/DetailsView/Body.js:100]: Can we move this block above export statement

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
Just minor changes otherwise looks good to me

**Pull request merged**